packages/utils/src/env.ts

```diff
@@ -1,24 +1,22 @@
 /**
- * This module mostly exists for optimizations in the build process
- * through rollup and terser.  We define some global constants which
- * are normally undefined.  However terser overrides these with global
- * definitions which can be evaluated by the static analyzer when
- * creating a bundle.
- *
- * In turn the `isDebugBuild` and `isBrowserBundle` functions are pure
- * and can help us remove unused code from the bundles.
+ * This module exists for optimizations in the build process through rollup and terser.  We define some global
+ * constants, which can be overridden during build. By guarding certain pieces of code with functions that return these
+ * constants, we can control whether or not they appear in the final bundle. (Any code guarded by a false condition will
+ * never run, and will hence be dropped during treeshaking.) The two primary uses for this are stripping out calls to
+ * `logger` and preventing node-related code from appearing in browser bundles.
  */
 
 declare const __SENTRY_BROWSER_BUNDLE__: boolean | undefined;
```

Comment:
Are we going to do the same for `__SENTRY_BROWSER_BUNDLE__`?

Reply:
Probably it makes sense to do so, yes. That's a future-me problem, though. 🙂